### ISSUE TITLE
is there a breaking change in v4 ?

### ISSUE THREAD

**Issue by rafipiccolo** [mailjet/mailjet-apiv3-nodejs#205]
its not clear from the changelog.

**Comment by scroll17** [comment]
Hey @rafipiccolo 

Yes, the `v4` version made major changes, we follow the `SemVer` convention and tag it as the next major release, but these changes are not shown in the CHANGELOG for a reason that `standard-version` and `CHANGELOG` were added after major changes.  
The `CHANGELOG` will useful thing for the next releases but not for previous ones. 

One of the most important things that were added it's the change single static method `connect` to both different methods `apiConnect` and `smsConnect`. 
As well we remove some legacy code from the library such as callback in `.request` method and now allow to use only Promise. 

In summary, we rewrote this library to `TypeScript`, change file structure, make the code more modular, add a `99%` cover using unit tests, add `Webpack` for build library for `node` and `browser` and in general, we add some features and made library usage lighter.

You can visit the [docs folder](https://github.com/mailjet/mailjet-apiv3-nodejs/tree/master/docs) where you can find documentation that was built by `typedoc` and in this documentation, you can find a list of all properties and methods for each class and descriptions for them.

**Comment by rafipiccolo** [comment]
I changed the connect method to apiConnect and all seems to work. thx, gj

**Comment by Narretz** [comment]
What's the reason for not just updating the changelog manually for v4? That's really odd, and will likely cost many devs at least 10 minutes.
